app/models/communitarian/resolution.rb

```diff
@@ -39,7 +39,7 @@ def schedule_jobs(post)
     private
 
     def resolution?(post)
-      post.topic.custom_fields["is_resolution"]
+      post.topic.custom_fields["is_resolution"] && post.polls.size.nonzero?
```

Comment:
`.size.nonzero?` instead of `.present?` for performance sake

Reply:
@DmitryBarskov `post.polls.exists?` will be better